src/util/common.jl

```diff
@@ -1,39 +1,39 @@
-"combine data into multinetwork summary"
-function process_network(network_data)
+# "combine data into multinetwork summary"
```

Comment:
Why is this code commented out?  If it is no longer needed, just remove it.

Reply:
it was previously used in the heuristic rop function, but a better implementation exists in iterative restoration.  I will remove it.